js/search.js

```diff
@@ -7,6 +7,26 @@ document.getElementById('search_query_query').addEventListener('keydown', functi
     }
 });
 
+// Add accessibility functionality:
+// "Press '/' to focus the searchbar".
+document.addEventListener('keydown', function (e) {
+    if (e.code && e.code !== 'Slash') {
+        return;
+    }
+    var searchInput = document.getElementById('search_query_query');
+    // Just ignore if we can't find the search input for some reason maybe we are on a page without it.
+    if (!searchInput) {
+        return;
+    }
+    // If we already have input focus ignore.
+    if (document.activeElement.tagName === 'INPUT') {
```

Comment:
Not sure if it's relevant but may want to exclude `input type="button"` so that buttons are treated the same whether the use an input or a button tag.

Reply:
Not sure I follow?

Assume you mean adding something like `&& document.activeElement.getAttribute('type') !== 'button'` so if the focus is on a button it can still be moved to the search?

That is a good idea if so, It's a shame so much of the web's javascript is obfuscated. I was trying to steal an existing implementation  but even looking at the list of keydown events registered on Github or any google site doesn't show anything for `/` despite supporting the functionality. I wonder if there is a nicer API etc for this, I don't do a lot of JS and have a sneaking feeling they've got some other API or maybe its just the fact that its a million pollyfills of abstraction away.